src/main/java/com/google/cloud/tools/libraries/json/CloudLibrary.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.libraries.json;
+
+import com.google.gson.Gson;
+import java.util.List;
+import javax.annotation.Nullable;
+
+/** Holds metadata about a single Cloud Library. */
+public final class CloudLibrary {
+
+  private String name;
+  private String id;
+  private String documentation;
+  private String description;
+  private String icon;
+  private List<String> transports;
+  private List<CloudLibraryClient> clients;
+
+  /** Use {@link #parse(String)} instead. */
+  private CloudLibrary() {}
+
+  /**
+   * Parses the given JSON representation of a {@link CloudLibrary} and returns a new instance
+   * deserialized from the given JSON.
+   */
+  public static CloudLibrary parse(String json) {
```

Comment:
whats the use of this method in practice? Given the json file, wouldn't the user first parse it into a `List<CloudLibrary>`?

Reply:
Yeah that's true, my original intention was to provide a way to parse a single `CloudLibrary` so it's not strictly limited to the existing content of the `libraries.json` file, but it really isn't necessary (and it's already pretty limited to the `libraries.json` anyway).

PTAL, I've removed this method.